src/components/filters/BooleanFilter.vue

```diff
@@ -0,0 +1,124 @@
+<template>
+  <label class="slider-checkbox-container">
+    <span
+      v-if="falseLabel"
+      class="label-text"
+    >{{ falseLabel }}</span>
+    <div class="slider-checkbox">
+      <div class="inner-slider">
+        <input
+          v-model="selection"
+          type="checkbox"
+        >
+        <div class="slider-knob bg-light" />
+        <div
+          class="slider-background"
+          :class="isSet ? 'bg-primary' : 'bg-secondary'"
```

Comment:
This filter is always set to a value. Just by adding the filter things wil be filtered. All other filters have a default state of not filtering the results.